tests/02as-info.pl

```diff
@@ -0,0 +1,25 @@
+push our @EXPORT, qw( AS_INFO );
+
+sub gen_token
+{
+   my ( $length ) = @_;
+   return join "", map { chr 64 + rand 63 } 1 .. $length;
+}
+
+struct ASInfo => [qw( localpart user_id as2hs_token hs2as_token path )];
```

Comment:
Throw in an id, because ASes have IDs now (even if they're not quite mandatory yet)